omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -198,16 +198,19 @@
 				</tr>
 			</c:if>
 			<tr><td colspan="2">&nbsp;</td></tr>
-			
+			<tr>
+				<openmrs:extensionPoint pointId="org.openmrs.locations" type="html">
```

Comment:
The extension point name I think should not suggest that it's for locations because not everyone will use it for locations.
Please use a more general purpose name like org.openmrs.userForm.extenstion1

Reply:
Did you also test that if no extensions are found, there is no blank area on the page in the UI?

Reply:
If datafilter module is not present then there is a small block of blank space. Attaching the screenshot for reference

![image](https://user-images.githubusercontent.com/40290306/77884752-4f9a0e80-7283-11ea-9e2f-3464aba12c35.png)

Reply:
>The extension point name I think should not suggest that it's for locations because not everyone will use it for locations.
Please use a more general purpose name like org.openmrs.userForm.extenstion1

How about `org.openmrs.userForm.custom.extension`